setup.py

```diff
@@ -33,7 +33,7 @@
     ]
 
     doc_requirements = [
-        "sphinx",
+        "sphinx<7.0",
```

Comment:
Why <7.0?

Reply:
There is a known bug with sphinx>7 and the sphinx rtd theme - https://github.com/readthedocs/readthedocs.org/issues/10279